net/dawn/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# This software is licensed under the Public Domain.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dawn
+PKG_VERSION:=2020-03-23
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/berlin-open-wireless-lab/DAWN.git
+PKG_SOURCE_VERSION:=42be2efdaaef0f8e7d204f558610a494577b14ec
+PKG_MIRROR_HASH:=aa3c5970cd853bd4db4ffa3868f7c1bfd0ad47f02a97a31fc093e4d24e75512d
+
+PKG_MAINTAINER:=Nick Hainke <vincent@systemli.org>
+PKG_LICENSE:=GPL-2.0
```

Comment:
This is using deprecated [SPDX License Identifier](https://spdx.org/licenses/).

Reply:
More specificallty, this should be -or-later or -only.

Reply:
I have no idea, what a good license is. Any recommendations?